samples/qml/view.qml

```diff
@@ -4,11 +4,14 @@
 
 import QtQuick 2.0
 import QtQuick.Controls 2.0
+
+// These are all used inside glean.js
 import QtQuick.LocalStorage 2.0
+import QtQml 2.15
```

Comment:
Does this mean Glean.js users for QML need to import Glean's dependencies manually? We should probably find a way around this in the final design.

Reply:
Yes. I believe the workaround for this would be to have the generated code for QML actually be a `metrics.qml` file, which imports the QML dependencies and all that. Not sure how that would work though, but I am sure there is a way around it.

Reply:
Ok, let's keep track of this, too.